### PR TITLE
feat: support formats option for multirendertarget

### PR DESCRIPTION
**Pull request by newbeea**
Add support for setting formats option of multirendertarget

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/13678/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13678/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/13678/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/13678/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/13678/merge
https://gui.babylonjs.com/?snapshot=refs/pull/13678/merge
https://nme.babylonjs.com/?snapshot=refs/pull/13678/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/13678/merge#BCU1XR#0

File: packages/dev/core/src/Materials/Textures/multiRenderTarget.ts [hunk 1]
```diff
@@ -491,14 +504,15 @@ export class MultiRenderTarget extends RenderTargetTexture {
...
         const layerIndex: number[] = [];
         const layerCounts: number[] = [];
 
         this._textureNames = textureNames;
 
-        this._initTypes(count, types, samplingModes, useSRGBBuffers, targetTypes, faceIndex, layerIndex, layerCounts, options);
+        this._initTypes(count, types, samplingModes, useSRGBBuffers, formats, targetTypes, faceIndex, layerIndex, layerCounts, options);
         this._multiRenderTargetOptions.types = types;
         this._multiRenderTargetOptions.samplingModes = samplingModes;
         this._multiRenderTargetOptions.useSRGBBuffers = useSRGBBuffers;
```
Comment (Popov72) [inline packages/dev/core/src/Materials/Textures/multiRenderTarget.ts:518]: `this._multiRenderTargetOptions.formats = formats;` is missing.

**Review by Popov72 — CHANGES_REQUESTED** [review]
There's still `Engines/RenderTargetWrapper._cloneRenderTargetWrapper` to update and we are good to go!

File: packages/dev/core/src/Materials/Textures/multiRenderTarget.ts (hunk 1 above)
Comment (newbeea) [reply packages/dev/core/src/Materials/Textures/multiRenderTarget.ts:518]: my bad

**Review by sebavan — COMMENTED** [review]
LGTM, I ll merge after a final @Popov72 review

**Comment by Popov72** [comment]
There's:

> There's still Engines/RenderTargetWrapper._cloneRenderTargetWrapper to update and we are good to go!

And it will be ok!

**Comment by newbeea** [comment]
> 

Sorry I didn't notice that, I have commit that now.

**Comment by sebavan** [comment]
Looks perfect to me, pinging @Popov72 just to be sure :-)

**Review by Popov72 — APPROVED** [review]

**Pull request merged**